src/patternfly/components/ChipGroup/docs/chip-group-multi-select.md

```diff
@@ -11,5 +11,5 @@ A chip-group used in a multi-select do not require a modifier as its parent will
 
 | Class | Applied To | Outcome |
 | -- | -- | -- |
-| `.pf-c-chip-group` | `<div>` | Initiates the container used to group chips. **Required.** |
+| `.pf-c-chip-group` | `<ul>` | Initiates the container used to group chips. **Required.** |
 | `.pf-c-button` | `<button>` | Initiates the button used to expand overflown chips. |
```

Comment:
should we document `.pf-c-chip` in this and/or the chip group toolbar docs?

Reply:
I've added one line here referencing it. I don't think we need to add all the docs in there. It should be pretty clear that the chip is required. I like having just chip group docs here. What do you think?

Reply:
Ah yeah, it's fine, I don't know what I was thinking :)